docs/man/git-lfs.1.ronn

```diff
@@ -33,8 +33,6 @@ commands and low level ("plumbing") commands.
     Display the Git LFS environment.
 * git-lfs-checkout(1):
```

Comment:
You get the same warning message for `git lfs checkout`. Maybe remove this one, too?